build-system/babel-config/minified-config.js

```diff
@@ -59,20 +59,11 @@ function getMinifiedConfig() {
     ],
     ['@babel/plugin-transform-for-of', {loose: true, allowArrayLike: true}],
```

Comment:
we should remove all the downleveling, like this `for-of` as well. It is currently separate so we can configure its opts.